gradle/libs.versions.toml

```diff
@@ -58,7 +58,7 @@ mockito = "5.5.0"
 mongo = "4.8.2"
 netty = "4.1.104.Final"
 newrelic-api = "5.14.0"
-okhttp = "5.0.0-alpha.11" # TODO is compiling against an alpha version intentional?
+okhttp = "4.11.0"
```

Comment:
Downgraded from 4.12.0 to 4.11.0. This better matches the 5.0 alpha we were compiling against in terms of Kotlin dependencies.